pgmpy/models/BayesianModel.py

```diff
@@ -442,6 +442,12 @@ def get_independencies(self, latex=False):
                     if independent_variables:
                         independencies.add_assertions([start, independent_variables,
                                                        observed])
+            local_independence = self.local_independencies(start)
+            common_assertions = set(local_independence.get_assertions())
+            & set(independencies.get_assertions())
```

Comment:
@kshitij10496 This is not a valid syntax. travis CI was failing because of this.